examples/passport-login/oauth2-providers.json

```diff
@@ -0,0 +1,47 @@
+{
+    "facebook-login": {
+        "provider": "facebook",
+        "module": "passport-facebook",
+        "profileFields": ["gender", "link", "locale", "name", "timezone",
+        "verified", "email", "updated_time", "displayName", "id"],
+        "clientID": "{facebook-client-id}",
+        "clientSecret": "ad4315e3453b012f6e4b8dd4bc1c0ae6",
```

Comment:
Please remove the `clientSecret`.

Reply:
@raymondfeng agreed , modified the configurations now